frontend/src/index.js

```diff
@@ -1,5 +1,106 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
+import ReactDOM from "react-dom";
+import React, { Component } from "react";
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import "normalize.css";
```

Comment:
Is this still a thing? Doesn't the Ant framework handle it for us?

Reply:
I forgot to remove normalize.css :stuck_out_tongue: